atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockTablesIntegrationTest.java

```diff
@@ -91,33 +104,59 @@ public void multipleSchemaMutationLockTablesObjectsShouldReturnSameLockTables()
     }
 
     @Test
-    public void shouldCreateLockTablesStartingWithCorrectPrefix() throws TException {
+    public void shouldCreateLockTablesWithCorrectName() throws TException {
+        lockTables.createLockTable();
+
+        assertThat(Iterables.getOnlyElement(lockTables.getAllLockTables()).getTablename(), equalTo("_locks"));
+    }
+
+    @Test
+    public void shouldCreateLockTableWithFixedCfId() throws Exception {
```

Comment:
Feels like this test is doing a lot of work just to check that we calculate the UUID using `UUID.nameUUIDFromBytes(fullTableName.getBytes())`.  This is testing the implementation rather than the behaviour so I'm not sure of the value.